src/main/java/org/owasp/esapi/PropNames.java

```diff
@@ -111,6 +111,7 @@ public final class PropNames {
     public static final String LOG_ENCODING_REQUIRED                                                            = "Logger.LogEncodingRequired";
     public static final String LOG_APPLICATION_NAME                                                             = "Logger.LogApplicationName";
     public static final String LOG_SERVER_IP                                                                    = "Logger.LogServerIP";
+    public static final String OMIT_EVENT_TYPE_IN_LOGS                                                          = "Logger.OmitEventTypeInLogs";
```

Comment:
Convention consistency:
Suggest following the existing naming scheme:
`LOG_EVENT_TYPE`

Maintains consistency with the positive action verbiage used in the other properties.

This will likely mean you'll need to change all of the booleans in the tests as well, but it will be more consistent with the existing convention.

Reply:
The static `String` is generally based on the last portion of the property name, so I think he is consistent given the name of the property. (And I suggested the property name in a GitHub issue #811 and in the discuss that it references (#810). So, my bad for that.

I'm fine if we want to rename the property to match the rest of these **Logger**  related properties, but I will leave it to you to suggest a property name them. I figured that if we called it the more conventional **Logger.LogOmitEventType** that developers may incorrectly infer from that that they could omit `Logger.EVENT_TYPE` when invoking the logger and **Logger.LogOmitEventTypeInLogs** almost sounded redundant, so I just dropped the initial 'Log' when I proposed a name. I do think this is consistent here given how the property is currently named though, so I suggest we leave this as-is here or else change the property name. And, as mentioned, I'm fine with either of you proposing a different property name.